src/main/java/org/stroganov/dao/SoapServiceLibraryDAO.java

```diff
@@ -0,0 +1,195 @@
+package org.stroganov.dao;
+
+import jakarta.xml.ws.WebServiceClient;
+import org.stroganov.entities.*;
+import org.stroganov.wsClient.BookArray;
+import org.stroganov.wsClient.IOException_Exception;
+import org.stroganov.wsClient.LibraryServerServiceImplService;
+import org.stroganov.wsClient.LibraryService;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+
+@WebServiceClient(name = "Client", targetNamespace = "http://example.org")
+public class SoapServiceLibraryDAO implements LibraryDAO {
```

Comment:
Rename and move to client package 
```suggestion
public class LibraryWebServiceClient implements LibraryDAO {
```
DAO - it's more about access object from data base

Reply:
Moved